src/Microsoft.PowerShell.Commands.Diagnostics/PdhHelper.cs

```diff
@@ -545,7 +545,7 @@ public uint ConnectToDataSource(StringCollection blgFileNames)
             string doubleNullTerminated = string.Empty;
             foreach (string fileName in blgFileNames)
             {
-                doubleNullTerminated += fileName + '\0';
+                doubleNullTerminated += fileName + "\0";
             }
```

Comment:
Shouldn't StringBuilder be preferred in loops instead of concat?

Reply:
It is not hot path.